release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -81,6 +81,27 @@ Release Team selection should happen in accordance with the [Release Team select
 ### Lead Shadows
 
 Once you have selected your Lead Shadows, each Shadow should get assigned to support two teams. One should be a team they have previously led or have extensive experience in. The other will be a team they have had no exposure to prior. They will work closely with their subteams to provide timely support. This will make sure that they are well-rounded to step up to lead.
+As the communication between Lead Shadows and Subteam Leads happens without the Release Team Lead, it is important for the Release Team Lead to check in with the Lead Shadows as well as the Subteam Leads to ensure that everything is running smoothly.
+
+Lead Shadows are also required to assist in various subteam tasks (as a secondary reviewer after the subteam review), including but not limited to:
+
+- Documentation:
+  - Review Release Notes PRs for clarity and completeness.
+  - Branch Sync PRs are created timely to prevent branch skew.
+  - Assisting the documentation subteam before the docs freeze.
+- Communications:
+  - Reviewing at least mid cycle blog and the release blog.
+- Enhancements:
+  - Assisting the enhancements subteam before PRR, Enhancements, and Code+Test Freeze.
+- Release Signal:
+  - Sporadically monitoring the release signal dashboard and alerting the Release Team Lead of any issues.
+
+The Release Lead Shadows are also required to assist in the following Release Team Lead tasks, including but not limited to:
+- Hosting the Release Team meetings (see [Release Team Meeting Host Playbook](#release-team-meeting-host-playbook) for more details)
+- Suggesting KEPs for mid cycle blog and release blog to the Release Team Lead and the Communications Lead.
+- Coordinating any freeze requests and starting a discussion in #sig-release Slack channel.
+
+The Release Team Lead should explicitly assign above-mentioned and additional tasks to Lead Shadows as needed.
```

Comment:
The idea behind this is to better formalise the tasks that lead shadows need to do. These are just some of the main tasks hence the wording `including but not limited to`

Reply:
I like this as it helps make more clear what we're trying to look out for from the lead perspective with the release team meetings, and how Release Lead Shadows can fill the gaps to help. Also making more concrete what  Release Lead Shadows will usually be asked to do.

This seems reasonable to me. @katcosgrove what do you think? Does any of this create a too many cooks in the kitchen situation?